backend/airbnb/__init__.py

```
@@ -1,10 +1,16 @@
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 
 import logging
+if not hasattr(settings, 'ENCRYPT_KEY'):
+    raise ImproperlyConfigured("Encrypt key must be in settings")
 
-cipher = Fernet(settings.ENCRYPT_KEY.encode())
+try:
+	cipher = Fernet(settings.ENCRYPT_KEY.encode())
+except Exception as e:
+    raise(f"Invalid encryption format: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)